src/compressed_tensors/quantization/lifecycle/apply.py

```diff
@@ -63,10 +64,16 @@
 _LOGGER = logging.getLogger(__name__)
 
 
-def load_pretrained_quantization(model: Module, model_name_or_path: str):
+def load_pretrained_quantization_inputs_outputs(
+    model: Module,
+    model_name_or_path: Optional[str] = None,
+    load_weight_quantization: Optional[bool] = False,
+):
     """
     Loads the quantization parameters (scale and zero point) from model_name_or_path to
-    a model that has already been initialized with a quantization config
+    a model that has already been initialized with a quantization config.
+
+    NOTE: only loads inputs/output parameters. Weight parameters are loaded by the decompressor
```

Comment:
nit: This is conditioned on the `load_weight_quantization` parameter

Reply:
The name is a bit of a misnomer if we still want to maintain the capability to load weight parameters